chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/CertifiedProductDetailsManager.java

```diff
@@ -277,9 +269,9 @@ public CertifiedProductSearchDetails getCertifiedProductDetailsBasic(Long certif
                 dto.getId(), true);
 
         CertifiedProductSearchDetails searchDetails = getCertifiedProductSearchDetails(dto);
+        List<SvapCriteriaMap> svapCriteriaMap = svapDao.getAllSvapCriteriaMap();
```

Comment:
Does this need to be called every time we get details for a listing? Does this data change often? I'm curious why you chose to move it out of the init method here. If it does change only occasionally, I wonder if we could do something with Ehcache instead?

Reply:
The data changes when an SVAP is editted/added/removed.  Not very often.  But if a user makes a change, this does not get updated.  We cannot call the `init` method when we change the SVAP because of  the multiple servers.  If we make the change on Server1, we cannot call the `init` method on Server2.
This pained me, too!